### PR TITLE
Add documentation for new Rabbit Air integration

### PR DESCRIPTION
**Pull request by al-s**
## Proposed change

Add documentation for the new RabbitAir integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66130
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3150
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @al-s, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21560).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/rabbitair.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: Rabbit Air
+description: Instructions on how to integrate Rabbit Air air purifier within Home Assistant.
+ha_category:
+  - Fan
+ha_iot_class: Local Polling
+ha_release: 2022.3
```
Comment (frenck) [inline source/_integrations/rabbitair.markdown]:
```suggestion
ha_release: 2024.2
```

File: source/_integrations/rabbitair.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ha_platforms:
+  - fan
+---
+
+The Rabbit Air integration allows you to control your air purifier over the local network. The following device models are currently supported:
+
+- MinusA2 (2-nd generation)
+- A3
+
+The fan platform of this integration allows you to turn on/off the unit, select the preset mode or set the speed manually.
```
Comment (frenck) [inline source/_integrations/rabbitair.markdown]:
```suggestion
The Rabbit Air integration lets you control your air purifier over the local network. The following device models are currently supported:

- MinusA2 (2-nd generation)
- A3

The fan platform of this integration allows you to turn the unit on/off, select the preset mode, or set the speed manually.
```

File: source/_integrations/rabbitair.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Retrieving the Access Token
```
Comment (frenck) [inline source/_integrations/rabbitair.markdown]:
```suggestion
## Pre-requisites
```

File: source/_integrations/rabbitair.markdown [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+To set up the integration, you will need to know a device address and an access token.
+
+1. Open the Rabbit Air mobile app. You will see a list of devices connected to your account.
+2. Tap the list item and the device control page will open.
+3. On the device page tap the `Edit` button. You will see a page with the device location and name settings.
+4. On this page, quickly tap on "Serial Number" several times until you see two more lines that were previously hidden. The first is the device ID, and the second is the access token.
```
Comment (frenck) [inline source/_integrations/rabbitair.markdown]:
```suggestion
1. Open the Rabbit Air mobile app. You will see a list of devices connected to your account.
2. Tap the list item, and the device control page will open.
3. On the device page, select the **Edit** button. You will see a page with the device location and name settings.
4. On this page, quickly tap **Serial Number** several times until you see two more previously hidden lines. The first is the device ID, and the second is the access token.
```

File: source/_integrations/rabbitair.markdown [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+For example, you got:
+
+<p class='img'>
+  <a href='/images/integrations/rabbitair/access_token.png' target='_blank'>
+    <img height='460' src='/images/integrations/rabbitair/access_token.png' alt='Screenshot: Access token on the &quot;Edit device&quot; screen'>
+  </a>
+</p>
+
+Then you can use `abcdef1234_123456789012345678.local` as the "Host" and `0123456789ABCDEF0123456789ABCDEF` as the "Access Token".
```
Comment (frenck) [inline source/_integrations/rabbitair.markdown]:
```suggestion
Then you can use `abcdef1234_123456789012345678.local` as the **Host** and `0123456789ABCDEF0123456789ABCDEF` as the **Access Token**.
```

**Review by frenck — DISMISSED** [review]

File: source/_integrations/rabbitair.markdown [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+The Rabbit Air integration lets you control your air purifier over the local network. The following device models are currently supported:
+
+- MinusA2 (2-nd generation)
+- A3
+
+The fan platform of this integration allows you to turn the unit on/off, select the preset mode, or set the speed manually.
+
+## Pre-requisites
```
Comment (c0ffeeca7) [inline source/_integrations/rabbitair.markdown]:
```suggestion
## Prerequisites
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @al-s :+1:

**Pull request merged**